scripts/links_checker/test/command/links_checker_command_test.dart

```diff
@@ -95,6 +95,25 @@ void main() {
         verify(linksChecker.checkFiles([])).called(equals(1));
       },
     );
+
+    test(
+      ".run() excludes files from the analyze based on the given ignore parameter",
+      () {
+        final paths = ['file1', 'path/to/file2'];
+        final ignore = ['path/'];
+        when(fileHelperUtil.getFiles(any)).thenReturn([]);
+        when(argumentsParser.parseArgResults(any))
+            .thenReturn(LinksCheckerArguments(paths: paths, ignore: ignore));
+
+        final command = LinksCheckerCommand(
+          fileHelperUtil: fileHelperUtil,
+          argumentsParser: argumentsParser,
+        );
+        command.run();
+
+        verify(fileHelperUtil.getFiles(['file1'])).called(1);
```

Comment:
```suggestion
        verifyNever(
          fileHelperUtil.getFiles(argThat(contains('path/to/file2'))),
        );
```
Would it be more reliable?